tests/test_display.py

```diff
@@ -12,7 +13,8 @@
 def display(qtbot):
     display = TyphonDisplay()
     qtbot.addWidget(display)
-    return display
+    yield display
```

Comment:
pytest fixtures can be used as context managers without even a decorator? Neat!

Reply:
https://docs.pytest.org/en/latest/yieldfixture.html 😄